helm/dagger/values.yaml

```diff
@@ -3,11 +3,14 @@ fullnameOverride: ""
 
 engine:
   # Set engine.config if you needa custom Dagger Engine config file
-  # config: |
-  #   debug = true
-  #   insecure-entitlements = ["security.insecure"]
-  #   [registry."registry:5000"]
-  #     http = true
+  config: |
+    debug = true
+    insecure-entitlements = ["security.insecure"]
+    [registry."registry:5000"]
+      http = true
+    
+    [registry."docker.io"]
+      mirrors = ["mirror.gcr.io"]
```

Comment:
do you think should we flag this? It's generally a "nice to have" but I'm wondering if all users will want/need this defaults.

thoughts @schlapzz @cpdeethree ?

Reply:
I am not sure if it makes sense to turn this on by default. I think dagger engine config needs good documentation, but I think that maybe keeping things "safe" by hiding them behind a comment is the right way here. So what I would like to see is an example configmap with all of the default values specified here, plus other common engine configs, but all of them are commented out so that people understand that they are overriding the default behavior by uncommenting them and that this is the recommended pattern to override engine configs

Reply:
+1 to not changing the image defaults.

We should provide an example, for sure, but I would not change the image defaults.